site/public/js/simple-grading.js

```diff
@@ -679,7 +679,13 @@ function setupSimpleGrading(action) {
             else if (direction == "down") new_selector_array[2] += 1;
             else if (direction == "left") new_selector_array[3] -= 1;
             else if (direction == "right") new_selector_array[3] += 1;
-
+            
+            //scroll to leftmost position, if reach first item in col with arrowkey navigation.
+            if(direction == "left" && new_selector_array[3]==0){
+               var container = document.querySelector('.scrollable-table');
+                container.scrollLeft ="-30px";
```

Comment:
the css implement for the scrollable table is
 .scrollable-table {
    overflow-x: auto;
}
I think the reason caused the error is that the first five columns of the student's information(number, section, name, etc) have the property (position: sticky). and that will hold these information on top of the table above other input boxes. So, when using the arrow key to move to the left, the input boxes is actually shown at the leftmost but underneath the student's information. I'm not sure if this can be fixed through css. However, I'm trying to modify the javascript code I wrote early to make sure the table shows ever column even when not reaching the leftmost one.

Reply:
> Ideally we want to use the native browser behavior here rather than a JavaScript implementation for accessibility reasons. I would recommend as a first step looking into the CSS styling affecting the browser's default leftward scrolling of the table, and only if you determine it's not reasonably effective to modify that, then as a workaround fall back to scripting it.

the css implement for the scrollable table is
.scrollable-table {
overflow-x: auto;
}
I think the reason caused the error is that the first five columns of the student's information(number, section, name, etc) have the property (position: sticky). and that will hold these information on top of the table above other input boxes. So, when using the arrow key to move to the left, the input boxes is actually shown at the leftmost but underneath the student's information. I'm not sure if this can be fixed through css. However, I'm trying to modify the javascript code I wrote early to make sure the table shows ever column even when not reaching the leftmost one.